src/en/hentalk/src/eu/kanade/tachiyomi/extension/en/hentalk/HenTalkWebViewInterceptor.kt

```diff
@@ -0,0 +1,73 @@
+package eu.kanade.tachiyomi.extension.en.hentalk
+
+import android.app.Application
+import android.os.Handler
+import android.os.Looper
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import okhttp3.Interceptor
+import okhttp3.Response
+import org.jsoup.Jsoup
+import uy.kohesive.injekt.Injekt
+import uy.kohesive.injekt.api.get
+import java.io.IOException
+import java.util.concurrent.CountDownLatch
+
+class HenTalkWebViewInterceptor : Interceptor {
+
+    override fun intercept(chain: Interceptor.Chain): Response {
+        val request = chain.request()
+        val response = chain.proceed(request)
+
+        if (response.headers("Content-Type").any { it.contains("text/html") }) {
```

Comment:
There's only one `Content-Type` header. If the site is sending multiple, they have an issue.

```suggestion
        if (response.headers["Content-Type"]!!.contains("text/html")) {
```